modules/plugins/languages/typst.nix

```diff
@@ -21,7 +21,10 @@
       lspConfig = ''
         lspconfig.typst_lsp.setup {
           capabilities = capabilities,
-          on_attach = default_on_attach,
+          on_attach = function(client, bufnr)
+            -- Disable semantic tokens if the issue persists
```

Comment:
put a more meaningful comment, something like "workaround for semantic tokens breaking on non-english characters"